Once bug 976205 has landed, the following error will become an orange:

08:14:48  WARNING -  TEST-UNEXPECTED-FAIL | None | A promise chain failed to handle a rejection: no user is logged in - rejection date: Fri Apr 04 2014 08:14:48 GMT-0700 (PDT) - See following stack:
08:14:48     INFO -  JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: PendingErrors.register :: line 164
08:14:48     INFO -  JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: this.PromiseWalker.completePromise :: line 670
08:14:48     INFO -  JS frame :: resource://gre/modules/services-sync/browserid_identity.js :: this.BrowserIDManager.prototype.initializeWithCurrentIdentity/< :: line 195
08:14:48     INFO -  JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: Handler.prototype.process :: line 863
08:14:48     INFO -  JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: this.PromiseWalker.walkerLoop :: line 742
08:14:48     INFO -  JS frame :: resource://services-common/async.js :: waitForSyncCallback :: line 99
08:14:48     INFO -  JS frame :: resource://services-common/async.js :: makeSpinningCallback/callback.wait :: line 141
08:14:48     INFO -  JS frame :: resource://services-sync/status.js :: this.Status._authManager :: line 36
08:14:48     INFO -  JS frame :: resource://gre/modules/services-sync/service.js :: onStartup :: line 310
08:14:48     INFO -  JS frame :: resource://gre/modules/services-sync/service.js :: <TOP_LEVEL> :: line 1568
08:14:48     INFO -  JS frame :: /builds/slave/test/build/tests/xpcshell/tests/services/sync/tests/unit/test_load_modules.js :: run_test :: line 47
08:14:48     INFO -  JS frame :: /builds/slave/test/build/tests/xpcshell/head.js :: _execute_test :: line 386

Once bug 976205 has landed, the following error will become an orange:

08:14:48  WARNING -  TEST-UNEXPECTED-FAIL | None | A promise chain failed to handle a rejection: no user is logged in - rejection date: Fri Apr 04 2014 08:14:48 GMT-0700 (PDT) - See following stack:
08:14:48     INFO -  JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: PendingErrors.register :: line 164
08:14:48     INFO -  JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: this.PromiseWalker.completePromise :: line 670
08:14:48     INFO -  JS frame :: resource://gre/modules/services-sync/browserid_identity.js :: this.BrowserIDManager.prototype.initializeWithCurrentIdentity/< :: line 195
08:14:48     INFO -  JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: Handler.prototype.process :: line 863
08:14:48     INFO -  JS frame :: resource://gre/modules/Promise.jsm -> resource://gre/modules/Promise-backend.js :: this.PromiseWalker.walkerLoop :: line 742
08:14:48     INFO -  JS frame :: resource://services-common/async.js :: waitForSyncCallback :: line 99
08:14:48     INFO -  JS frame :: resource://services-common/async.js :: makeSpinningCallback/callback.wait :: line 141
08:14:48     INFO -  JS frame :: resource://services-sync/status.js :: this.Status._authManager :: line 36
08:14:48     INFO -  JS frame :: resource://gre/modules/services-sync/service.js :: onStartup :: line 310
08:14:48     INFO -  JS frame :: resource://gre/modules/services-sync/service.js :: <TOP_LEVEL> :: line 1568
08:14:48     INFO -  JS frame :: /builds/slave/test/build/tests/xpcshell/tests/services/sync/tests/unit/test_load_modules.js :: run_test :: line 47
08:14:48     INFO -  JS frame :: /builds/slave/test/build/tests/xpcshell/head.js :: _execute_test :: line 386
Created attachment 8402016
Fixing the issue

I took the opportunity to fix a few other uncaught async errors due to typoes.
Comment on attachment 8402016
Fixing the issue

Review of attachment 8402016:
-----------------------------------------------------------------

r? for mhammond for browserid_identity.
https://hg.mozilla.org/integration/fx-team/rev/3f1abbf928c3
https://hg.mozilla.org/mozilla-central/rev/3f1abbf928c3


Dozens of  TEST-UNEXPECTED-FAIL | None | A promise chain failed to handle a rejection: no user is logged in



Bug 992363 - Fix a few uncaught async errors in services. r=rnewman 